tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1160,7 +1163,9 @@ ROOT::Experimental::Internal::RNTupleFileWriter::Recreate(std::string_view ntupl
    FILE *fileStream = fopen(std::string(path.data(), path.size()).c_str(), "wb");
 #endif
 #endif
-   R__ASSERT(fileStream);
+   if (!fileStream) {
+      throw RException(R__FAIL("f*open*(" + std::string(path) + ", *, \"wb\") failed with errno: " + strerror(errno)));
```

Comment:
I think this `"f*open* [...]"` is not very understandable: if this is supposed to be a user-facing message I think I'd not go into that much detail: I'd stick with a `"fopen(" + string(path) + ", \"wb\") failed with errno [...]"`